go/vendor/vendor.json

```diff
@@ -602,16 +602,34 @@
 			"revisionTime": "2018-03-22T21:26:26Z"
 		},
 		{
-			"checksumSHA1": "mGbTYZ8dHVTiPTTJu3ktp+84pPI=",
+			"checksumSHA1": "nj123dGxKPIF17B6y3VEK2pihRk=",
```

Comment:
don't think you need to pull in the root `testify` package either (it has no go code)

Reply:
ah ok let me try that again. I tried update just require at first and then it failed to compile because I think it calls into assert. thanks!